CSharp/BatchExplorer/Views/MainView.xaml

```diff
@@ -1085,6 +1085,12 @@
                                 <DataGrid ItemsSource="{Binding Path=SelectedTask.OutputFiles}"
                                           SelectedItem="{Binding Path=SelectedTask.SelectedTaskFile}"
                                           Visibility="{Binding Path=SelectedTask.HasOutputFiles, Converter={StaticResource BVC}, FallbackValue=Hidden}">
+                                    <DataGrid.InputBindings>

```

Comment:
For consistency, can you also add this change to the `SelectedComputeNode.Files` section on line `1292`?